src/changelog.rs

```diff
@@ -71,7 +71,7 @@ impl<'a> Changelog<'a> {
 
         let mut log = tera.render("template", &context)?;
         if let Some(metadata) = &self.config.metadata {
-            log.push_str(&format!("\n{}\n", metadata));
+            log.push_str(&format!("\n\n{}\n", metadata));
```

Comment:
I'm a bit confused here, without this the diff says one newline has been removed, but I don't see how my code removes more newlines than the old code. When explicitly adding this newline in the config parsing instead of adding it here, the diff says there's an extra newline. I'm really a bit lost here.